utils/format/balance.ts

```diff
@@ -88,6 +88,10 @@ export function roundTo(value: number | string, limit = 2) {
   })
 }
 
+export function formatBalanceWithBlank(value: number, decimals?: number, chainSymbol?: string) {
```

Comment:
fyi: even though format has default `decimals` of `12` but not all chains have the same value